pkgs/development/libraries/quarto/default.nix

```diff
@@ -0,0 +1,67 @@
+{ stdenv
+, lib
+, pandoc
+, esbuild
+, deno
+, fetchurl
+, nodePackages
+, makeWrapper
+}:
+
+stdenv.mkDerivation rec {
+  pname = "quarto";
+  version = "1.0.38";
+
+  src = fetchurl {
+    url = "https://github.com/quarto-dev/quarto-cli/releases/download/v${version}/quarto-${version}-linux-amd64.tar.gz";
+    sha256 = "sha256-bqEGRX6ihyrgmIxqkV9jr/2QfWM5LWnsjoGpTqK9qOc=";
+  };
+
+  buildInputs = [
+      pandoc
+      deno
+      esbuild
+      makeWrapper
```

Comment:
```suggestion
  nativeBuildInputs = [
    makeWrapper
```

Reply:
`pandoc`, `deno`, `esbuild` can be hooked into `QUARTO_*` environment variables, so I think they are not needed in buildInputs.